packages/tizen_bundle/README.md

```diff
@@ -0,0 +1,133 @@
+# tizen_bundle
+
+Tizen bundle APIs.
+
+## Usage
+
+To use this package, add `tizen_bundle` as a dependency in your `pubspec.yaml` file.
+
+```yaml
+depenedencies:
+  tizen_bundle:
+
```

Comment:
Please specify the package version (`^0.1.0`) and remove the unnecessary newline.

Reply:
Done.